src/hv_exc.c

```diff
@@ -38,6 +38,7 @@ static u64 exc_entry_time;
 
 extern u32 hv_cpus_in_guest;
 extern int hv_want_cpu;
+extern bool hv_want_cpu_in_rendezvous;
```

Comment:
Instead of doing it like this, how about we add a separate int called `hv_pinned_cpu`?